lsm-tree.cabal

```diff
@@ -34,7 +34,14 @@ common wno-x-partial
 common language
   default-language:   GHC2021
   default-extensions:
+    DeriveAnyClass
+    DerivingStrategies
+    DerivingVia
+    ExplicitNamespaces
```

Comment:
This one doesn't seem to be used anywhere.

Reply:
See docs for https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/control.html#extension-GHC2021

`ExplicitNamespaces` is transitively enabled by `TypeOperators`, but not by `GHC2021`.

Reply:
I see thanks.